### PR TITLE
max_length lints: Subject common_name, locality_name, organization_name, and state_name

### PR DESCRIPTION
**Pull request by jddicki2**
Continuation of PR#36 with fixes.

### CODE REVIEW

**Comment by zakird** [comment]
@kumarde please do first pass reivew.

File: lints/lint_subject_common_name_max_length.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	return nil
+}
+
+func (l *subjectCommonNameMaxLength) CheckApplies(c *x509.Certificate) bool {
+	// Add conditions for application here
+	return true
+}
+
+func (l *subjectCommonNameMaxLength) RunTest(c *x509.Certificate) (ResultStruct, error) {
+	if len(c.Subject.CommonName) <= 64 {
```
Comment (kumarde) [inline lints/lint_subject_common_name_max_length.go]: for clarity, can we have all the max_lengths follow 
```
if len(field) > value {
    return Error
}
return Pass
```

**Comment by kumarde** [comment]
This LGTM, @zakird?

File: lints/lint_subject_locality_name_max_length.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	return true
+}
+
+func (l *subjectLocalityNameMaxLength) RunTest(c *x509.Certificate) (ResultStruct, error) {
+	for _, j := range c.Subject.Locality {
+		if len(j) > 128 {
+			return ResultStruct{Result: Error}, nil
+		}
+	}
+
```
Comment (zakird) [inline lints/lint_subject_locality_name_max_length.go]: probably don't need the whitespace here.

**Comment by kumarde** [comment]
@zakird I would merge but I don't have write access

**Pull request merged**